modules/pages/client/components/Home.component.js

```diff
@@ -167,13 +179,13 @@ export default function Home({ user, isNativeMobileApp, photoCredits }) {
                 </Trans>
               </p>
             </div>
-            <div className="col-md-7">
-              <div className="home-browser">
-                <div className="home-browser-circle"></div>
-                <div className="home-browser-circle"></div>
-                <div className="home-browser-circle"></div>
-                <div className="home-browser-screenshot home-browser-screenshot-search"></div>
-              </div>
+            <div aria-hidden className="col-md-7">
```

Comment:
Hiding these images from screen readers since they're just for illustrative purposes.